jetbrains-ultimate/src/software/aws/toolkits/jetbrains/services/lambda/nodejs/NodeJsDebugUtils.kt

```diff
@@ -32,6 +33,9 @@ object NodeJsDebugUtils {
         debugPorts: List<Int>
     ): XDebugProcessStarter = object : XDebugProcessStarter() {
         override fun start(session: XDebugSession): XDebugProcess {
+            // mark the build folder as dirty so the debugger can find sourcemaps
+            VfsUtil.markDirty(true, true, VfsUtil.findFileByIoFile(state.builtLambda.templateLocation.parent.toFile(), true))
```

Comment:
Should we move this out of here and call it for all runtimes? We theoretically have the same problem with at least python

Reply:
Why mark dirty instead of LocalFileSystem.getInstance().refresh()?

Does marking dirty run the refresh immediately?

Reply:
Will take a look, the idea is to avoid a re-index while ensuring the node debugger finds/reads a fresh version of the source maps

Reply:
Both work, but I think we prefer the LocalFileSystem variant since it should be faster?